packages/codemod/plop/component/test.hbs

```diff
@@ -0,0 +1,19 @@
+import transform from "../{{kebabCase componentName}}-component-migration";
+import { defineInlineTest } from "jscodeshift/src/testUtils";
+
+function prependImport(source: string): string {
+  return `
+    import { {{pascalCase componentName}} } from "monday-ui-react-core";
+    ${source}
+  `;
+}
+
+describe("{{pascalCase componentName}} component migration", () => {
```

Comment:
Does it pass right after the generation?

Reply:
actually no, but maybe it's good that it won't pass, so it would attract attention for the plop user